ArduCopter/mode_rtl.cpp

```diff
@@ -342,19 +341,17 @@ void ModeRTL::descent_run()
     // set motors to full range
     motors->set_desired_spool_state(AP_Motors::DesiredSpoolState::THROTTLE_UNLIMITED);
 
-    // process roll, pitch inputs
-    loiter_nav->set_pilot_desired_acceleration(target_roll, target_pitch);
-
-    // run loiter controller
-    loiter_nav->update();
+    Vector2f accel;
```

Comment:
I think maybe we should just collapse these two lines together like this:

`pos_control->input_vel_accel_xy(vel_correction, Vector2f{});`

I think there is another place where we do the same thing that should be modified as well.  It doesn't really matter much but..